docs/guides/pull-request-title-generator.mdx

```diff
@@ -0,0 +1,65 @@
+---
+name: Pull Request Name Generator
+route: /pull-request-name-generator
+hidden: true
+---
+
+import { useState } from "react";
+import { Select, Option } from "@jobber/components/Select";
+import { Text } from "@jobber/components/Text";
+import { Card } from "@jobber/components/Card";
+import { Content } from "@jobber/components/Content";
+import { Divider } from "@jobber/components/Divider";
+import { Emphasis } from "@jobber/components/Emphasis";
+import { InputText } from "@jobber/components/InputText";
+
+# Pull Request Name Generator
+
+export function PRNameGenerator() {
+  const [type, setType] = useState("fix");
+  const [scope, setScope] = useState();
+  const [description, setDescription] = useState();
+  const [jira, setJira] = useState();
+  return (
+    <Card title="Tell us about your PR">
+      <Content spacing="large">
+        <Text>
+          I want to{" "}
+          <Select inline value={type} onChange={val => setType(val)}>
+            <Option value="fix">fix something that's broken</Option>
+            <Option value="feat">add some new functionality</Option>
+            <Option value="docs">write documentation</Option>
+            <Option value="build">improvements to the build system</Option>
```

Comment:
I want to `improve the build system`

Also got some feedback that "build system" is unclear...

Reply:
This isn't an option that should be often used. I also feel like if someone is doing this type of work they will know what this means. 

the build system refers to how to code is built/compiled.

Reply:
Good point! Yes the feedback was from a designer (albeit a Technical Designer) but if one of us is touching the build system that is probably a red flag 😹